susemanager/src/mgr-create-bootstrap-repo

```diff
@@ -1,41 +1,58 @@
 #!/usr/bin/python3
-from __future__ import print_function
+
 import sys
 import os
 import shutil
-import logging
 import subprocess
+import time
+import traceback
+from optparse import OptionParser
+from rhn import rhnLockfile
 from spacewalk.server import rhnSQL
-from uyuni.common import usix
+from spacewalk.common.rhnLog import initLOG, log_time, log_clean
 from spacewalk.common.rhnConfig import CFG, initCFG
-from optparse import OptionParser
-import textwrap
 from string import Template
-
-try:
-   input = raw_input
-except NameError:
-   pass
-
-msg_wrapper = textwrap.TextWrapper()
-msg_wrapper.initial_indent = '- '
-msg_wrapper.width = 75
-msg_wrapper.subsequent_indent = '  '
+from uyuni.common import usix, rhnLib
 
 sys.path.append("/usr/share/susemanager")
 
 initCFG('server.susemanager')
 rhnSQL.initDB()
 
 basepath = CFG.MOUNT_POINT or '/var/spacewalk'
+logfile = '/var/log/rhn/mgr-create-bootstrap-repo.log'
```

Comment:
As far as I can see, this log is now always generated.

IIUC, then I strongly suggest we add this info to the help of this script.

Reply:
ok, and I added the file to logrotate configuration